Content.Server/GameObjects/Components/Items/Storage/ItemComponent.cs

```diff
@@ -121,34 +74,5 @@ protected override void Activate(IEntity user, ItemComponent component)
                 }
             }
         }
-
-        public override ComponentState GetComponentState(ICommonSession session)
-        {
-            return new ItemComponentState(EquippedPrefix);
-        }
-
-        public void OnExplosion(ExplosionEventArgs eventArgs)
-        {
-            var sourceLocation = eventArgs.Source;
-            var targetLocation = eventArgs.Target.Transform.Coordinates;
-            var dirVec = (targetLocation.ToMapPos(Owner.EntityManager) - sourceLocation.ToMapPos(Owner.EntityManager)).Normalized;
-
-            float throwForce;
-
-            switch (eventArgs.Severity)
-            {
-                case ExplosionSeverity.Destruction:
-                    throwForce = 30.0f;
-                    break;
-                case ExplosionSeverity.Heavy:
-                    throwForce = 20.0f;
-                    break;
-                default:
-                    throwForce = 10.0f;
-                    break;
-            }
-
-            Owner.TryThrow(dirVec * throwForce);
-        }
```

Comment:
Why have you removed this completely?

Reply:
Though that was handled elsewhere, but I see it was not. Re-added explosions launching entities on a separate component.